contracts/drafts/meta-tx/GSNContext.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.5.0;
+
+import "./Context.sol";
+
+/*
+ * @dev Enables GSN support by recognizing calls from RelayHub and extracting
+ * the actual sender and call data from the received values.
+ */
+contract GSNContext is Context {
+    address private _relayHub;
+
+    constructor(address relayHub) public {
+        _setRelayHub(relayHub);
+    }
+
+    function _setRelayHub(address relayHub) internal {
+        _relayHub = relayHub;
+    }
+
+    // Currently required by the RelayProvider
```

Comment:
Is it planned for this to be removed?

Could we make this an `external` function?

Reply:
This is related to https://github.com/OpenZeppelin/openzeppelin-gsn-provider/issues/11